src/fal/packages/dependency_analysis.py

```diff
@@ -10,6 +10,24 @@
 import importlib_metadata
 
 
+def _is_fal_pre_release() -> bool:
+    from dbt.semver import VersionSpecifier
+
+    raw_fal_version = importlib_metadata.version("fal")
+    fal_version = VersionSpecifier.from_version_string(raw_fal_version)
+    return fal_version.prerelease
+
+
+def _get_fal_root_path() -> Path:
+    import fal
+
+    # If this is a development version, we'll install
+    # the current fal itself.
+    base_dir = Path(fal.__file__).parent.parent.parent
+    assert (base_dir / ".git").exists()
```

Comment:
hah is this a trick you have used before? maybe looks for a poetry related package?